albyhub/umbrel-app.yml

```
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: albyhub
+name: Alby Hub
+tagline: Your Hub to connect to the Bitcoin world
+category: finance
+version: "1.6.0"
+port: 59000
+description: >-
+  Alby Hub is your gateway to Bitcoin world and allows you to use your Umbrel lightning node with the growing number of exciting bitcoin and nostr applications.
+  Connect any bitcoin app to your Hub for self-custodial payments anywhere — websites, mobile apps, desktop apps, games, CLI tools and more.
+
+  Use Alby Hub to manage your lightning channels through one sleek, user-friendly interface.
+developer: Alby
+website: https://albyhub.com
+submitter: Alby
+submission: https://github.com/getumbrel/umbrel-apps/pull/1409
+repo: https://github.com/getAlby/hub
+support: https://github.com/getAlby/hub/issues
+gallery:
+  - 1.png
+  - 2.png
+  - 3.png
+  - 4.png
```

Can you please change to:
```suggestion
gallery:
  - 1.jpg
  - 2.jpg
  - 3.jpg
  - 4.jpg
```

Don't worry about converting the actual images, I'll do that when I commit them to https://github.com/getumbrel/umbrel-apps-gallery